src/main/java/firebase/sdk/dashboard/dao/PlatformReleaseDao.java

```diff
@@ -0,0 +1,51 @@
+package firebase.sdk.dashboard.dao;
+
+import firebase.sdk.dashboard.data.Release;
+import firebase.sdk.dashboard.data.SDKReleaseMetadata;
+import firebase.sdk.dashboard.data.Platform;
+import java.util.List;
+
+/**
+ * An interface for retrieving and manipulating data about platforms and SDK releases. 
+ */
+public interface PlatformReleaseDao {
+
+  /**
+   * Retrieves a list of platform objects from a data source.
+   *
+   * @return A list of objects representing all the platforms.
+   */
+  public List<Platform> getPlatforms();
+
+  /**
+   * Retrieves a list of the releases for the given platform.
+   *
+   * @param platform The platform the releases are a part of.
+   * @return A list of Release objects representing all the releases in that platform
+   * in descending order.
+   */
+  public List<Release> getPlatformReleases(Platform platform);
+
+  /**
+   * Adds the given release object to given platform in a data source. Only admins should
+   * be able to add release objects to data sources.
+   * 
+   * @param platform The platform the release is being added to.
+   * @param release The release object that is to be stored in the data source.
+   * @return Whether or not the release object was succesfully added to the 
```

Comment:
Not super clear how will you notify the caller of this method if it was a successful operation or not. Through exception I understand - add a TODO to define an exception.